deploy/vagrant/modules/mysql/templates/mysql_apt_config.dat.erb

```diff
@@ -0,0 +1,18 @@
+Name: mysql-apt-config/select-product
+Template: mysql-apt-config/select-product
+Value: Ok
+Owners: mysql-apt-config
+Flags: seen
+Variables:
+ selected_preview = Disabled
+ selected_server = mysql-8.0
+ selected_tools = Enabled
+
+Name: mysql-apt-config/select-server
+Template: mysql-apt-config/select-server
+Value: mysql-8.0
+Owners: mysql-apt-config
+Flags: seen
+Variables:
+ installed_server = MySQL Server 5.7
```

Comment:
I bet this is the bug that caused the init.d rough edge --- i'm bringing in a dpkg configuration that tells apt MySQL 5.7 is currently installed, onto a new container where nothing is currently installed.  So maybe i should simply remove the `installed_server` variable, or maybe i have to do something else.

(It *is* necessary to record and replay this apt config file, because under normal circumstances `dpkg -i /usr/local/src/mysql-apt-config_0.8.10-1_all.deb` (see shell script above) would prompt for settings, and precreating a config file is the correct way to avoid that.  But i suspect i recorded the wrong setting here, and that's why the init file wasn't created.)